docs/docs/components/api/README.md

```diff
@@ -199,14 +200,16 @@ Async options allow users to select prop values that can be programmatically-gen
 | ------------- | --------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `options()`   | `method`  | optional  | Typically returns an array of values matching the prop type (e.g., `string`) or an array of object that define the `label` and `value` for each option. The `page` and `prevContext` input parameter names are reserved for pagination (see below).<br>&nbsp;<br>When using `prevContext` for pagination, it must return an object with an `options` array and a `context` object with a `nextPageToken` key. E.g., `{ options, context: { nextPageToken }, }` |
 | `page`        | `integer` | optional  | Returns a `0` indexed page number. For use with APIs that accept a numeric page number for pagination.                                                                                                                                                                                                                                                                                                                                                         |
-| `prevContext` | `string`  | optional  | Return a string representing the context for the previous `options` execution. For use with APIs that accept a token representing the last record for pagination.                                                                                                                                                                                                                                                                                              |
+| `prevContext` | `string`  | optional  | Returns a string representing the context for the previous `options` execution. For use with APIs that accept a token representing the last record for pagination.                                                                                                                                                                                                                                                                                              |
+| `query` | `string`  | optional  | Returns a string with the user input if the prop has the `useQuery` property set to `true`. For use with APIs that return items based on a query or search parameter.                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| `query` | `string`  | optional  | Returns a string with the user input if the prop has the `useQuery` property set to `true`. Use with APIs that return items based on a query or search parameter.                                                                                                                                                                                                                                                                                              |
```

Reply:
Removed 'For use with' and replaced with 'Use with' for consistency with Google's Developer docs style guide (use active voice).